native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -2655,10 +2776,34 @@ static int proxy_cluster_trans(request_rec *r)
     proxy_server_conf *conf = (proxy_server_conf *)
         ap_get_module_config(sconf, &proxy_module);
 
-    proxy_vhost_table *vhost_table = read_vhost_table(r);
-    proxy_context_table *context_table = read_context_table(r);
-    proxy_balancer_table *balancer_table = read_balancer_table(r);
-    proxy_node_table *node_table = read_node_table(r);
+    proxy_vhost_table *vhost_table = cached_vhost_table;
```

Comment:
Should we lock around this with the table sets in the watchdog thread to ensure we're safely using one set of tables?